superset/assets/javascripts/explore/stores/visTypes.js

```diff
@@ -21,31 +25,37 @@ export const sections = {
     ],
   },
   colorScheme: {
-    label: 'Color Scheme',
+    label: ('Color Scheme'),
+    labelDisplay: t('Color Scheme'),
     controlSetRows: [
       ['color_scheme'],
     ],
   },
   sqlaTimeSeries: {
-    label: 'Time',
-    description: 'Time related form attributes',
+    label: ('Time'),
+    labelDisplay: t('Time'),
+    description: ('Time related form attributes'),
+    descriptionDisplay: t('Time related form attributes'),
     expanded: true,
     controlSetRows: [
       ['granularity_sqla', 'time_grain_sqla'],
       ['since', 'until'],
     ],
   },
   sqlClause: {
-    label: 'SQL',
+    label: ('SQL'),
+    labelDisplay: t('SQL'),
     controlSetRows: [
       ['where'],
       ['having'],
     ],
-    description: 'This section exposes ways to include snippets of SQL in your query',
+    description: ('This section exposes ways to include snippets of SQL in your query'),
+    descriptionDisplay: t('This section exposes ways to include snippets of SQL in your query'),
   },
   NVD3TimeSeries: [
     {
       label: 'Query',
```

Comment:
Other labels have parens now,out of curiosity why?

Reply:
Also why we are duplicating label -> labelDisplay amd description -> descriptionDisplay?